### ISSUE TITLE
I am not able to create embeddings

### ISSUE THREAD

**Issue by nickknyc** [bbawj/obsidian-semantic-search#6]
Running Windows 11 
Am able to generate input.csv (attached)
But not the embedding See attached 

![SemanticObsidianError](https://github.com/bbawj/obsidian-semantic-search/assets/1598572/56c428de-1e6f-4782-898a-913c5d6fa902)

[input.csv](https://github.com/bbawj/obsidian-semantic-search/files/12218090/input.csv)

**Comment by bbawj** [comment]
Ah I think in this case your input.csv included a line (row 13) with an empty string section and body, causing the request to fail. Let me create a workaround for this but in the meantime, you can delete that row in your input before generating embeddings and it should work.

**Comment by nickknyc** [comment]
Still having issues 
I removed line 13, but the same error came up - but it did create a embeddings file. 
I then deleted the input and embeddings files and replaced manifest.json and main.js with the 3.1 versions. I was able to generate the input and embeddings files with no errors, however when I tried a query i just get hung with this error
![semantic](https://github.com/bbawj/obsidian-semantic-search/assets/1598572/1c9a8a66-62ac-4d94-821f-776d2db7fb02)

**Comment by bbawj** [comment]
Could you open the developer console (Ctrl+Shift+I) and post the error message that you get?

**Comment by nickknyc** [comment]
I don't get an error now - are you asking me to look and see if there is an error?

**Comment by nickknyc** [comment]
Here is what I have

[obsidian.md-1690892106960.log](https://github.com/bbawj/obsidian-semantic-search/files/12229006/obsidian.md-1690892106960.log)
[obsidian.md-1690892246774.log](https://github.com/bbawj/obsidian-semantic-search/files/12229007/obsidian.md-1690892246774.log)

**Comment by bbawj** [comment]
Thanks for providing the logs!

From them it looks as though there might be some sort of invalid input between row 666 to 888 of your `input.csv`. Could you help to confirm if there are any entries where both the section and body are empty? In your previous attached `input.csv` it looked like rows 698, 710 and others are like that. Did generating input with version 0.3.1 clear those invalid rows?

The error you are facing with the query looks as though its because of a corrupted `embeddings.csv` file from the error:
`Embedding should be comma-separated list of f32`. 
This could be because the previous step to generate embeddings failed and hence it is not able to parse it correctly.

**Comment by nickknyc** [comment]
Hey Brendan,
I did find a bunch of blanks, I deleted those rows (using Excel) and am still having the same behavior - modal just hangs there.
While the log seems to say that all the embeddings were created ok, I still do not get a success message as I do with the create inputs function. Log Attached.
[obsidian.md-1690898232281 -3.log](https://github.com/bbawj/obsidian-semantic-search/files/12229993/obsidian.md-1690898232281.-3.log)
